dev/tasks/linux-packages/apache-arrow/yum/centos-8/Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN \
     glog-devel \
     gobject-introspection-devel \
     gtk-doc \
+    libarchive \
```

Comment:
We don't need this, right?

Reply:
Yes, will remove.

Reply:
We don't need this, could you revert this before we merge this?

Reply:
Thanks!

Reply:
Some explanation:

`libarchive` is pre-installed in `arm64v8/centos:8`, but not in `centos:8`.
This dockerfile is based on `arm64v8/centos:8`, so it's okay.
`verify-yum.sh` is based on `centos:8` (multi arch image) and fails.

Will we unify x86 and arm to use `centos:8` only?